File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.js
function solution(n) {
    let cnt = new Set();
    for(let i=1; i<=n; i++) {
        for(let j=2; j<i; j++) {
            if(i%j === 0) {
                cnt.add(i)
            }
        }
    }
    return cnt.size;
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.java
import java.util.*;

class Solution {
    public int solution(int n) {
        Set<Integer> cnt = new HashSet<>();

        for (int i = 1; i <= n; i++) {
            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    cnt.add(i);
                    break;
                }
            }
        }

        return cnt.size();
    }
}